### DISCUSSION TITLE
Support deploying on Fargate.

### DISCUSSION THREAD

**Discussion by davinchia** [airbytehq/airbyte#41085] [Platform Features or Ideas]
## Tell us about the problem you're trying to solve
Several users have asked about deploying to Fargate. This is a placeholder issue for the time being.

Linked to #4847

**Comment by pixie79** [comment]
Ideally need to deploy to Fargate as not allowed to run EKS, trying to only use server less components

**Comment by Mannyknp** [comment]
We would also prefer Fargate over EKS for airbyte deployment.

[a comment by an automated account is not shown]
